internal/app/machined/pkg/system/log/log.go

```diff
@@ -70,12 +69,6 @@ func (l *Log) Close() error {
 	return l.source.Close()
 }
 
-// Read implements chunker.Chunker.
-func (l *Log) Read(ctx context.Context) <-chan []byte {
```

Comment:
gc'ed unused method, and it was dangerous, as it used same `*os.File` as for writing to the logs, while `chunker` might still Seek the stream